src/components/WebChat/index.jsx

```diff
@@ -164,21 +186,21 @@ class WebChat extends React.Component {
     }
   }
 
-  resetInput() {
-    this.setState({
-      input: undefined,
-    });
-  }
-
   handleKeyPress(e) {
     if (e && e.nativeEvent.keyCode === 13) {
-      this.sendMessage(this.state.input);
+      this.sendMessage();
```

Comment:
You remove the parameter hier, but line 180 you add the parameter to function signature. Is it intended?